google/cloud/security/inventory/pipelines/load_firewall_rules_pipeline.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Pipeline to load firewall rules into Inventory.
+
+This pipeline depends on the LoadProjectsPipeline.
+"""
+
+import json
+
+from dateutil.parser import parse
+
+from google.cloud.security.common.gcp_api import errors as api_errors
+from google.cloud.security.common.util import log_util
+from google.cloud.security.inventory.pipelines import base_pipeline
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class LoadFirewallRulesPipeline(base_pipeline.BasePipeline):
+    """Load firewall rules for all projects."""
+
+    RESOURCE_NAME = 'firewall_rules'
+
+    def _transform(self, firewall_rules_map):
+        """Transform firewall rules map into loadable format for Cloud SQL.
+
+        Loading the project id as project number is not supported by the
+        GCP firewall api.
+
+        Args:
+            firewall_rules_map: A dict mapping projects with a list of their
+                firewall rules.
+                {project_id: [firewall_rules]}
+
+        Yields:
+            An iterable of loadable firewall rules as a per-firewall dictionary.
+        """
+        for project_id, firewall_rules in firewall_rules_map.iteritems():
+            for firewall_rule in firewall_rules:
+                # Drop the timezone which causes mysql to throw warning.
+                creation_time = parse(firewall_rule.get('creationTimestamp'))
```

Comment:
Can you leverage this util [function](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/common/util/parser.py#L49)? If not can you make it a util function?